snapcraft.yaml

```diff
@@ -1,79 +1,48 @@
 name: brave
 version: 1.18.70
-base: core18
+grade: stable
+architectures:
+  - build-on: amd64
 summary: Browse faster and safer with Brave.
 description: |
  The new Brave browser automatically blocks ads and trackers, making it
  faster and safer than your current browser. Start supporting publishers
  today with our new Brave Payments system.
-
-grade: stable
 confinement: strict
-
-architectures:
-  - build-on: amd64
-
-parts:
-  brave:
-    plugin: dump
-    source: https://github.com/brave/brave-browser/releases/download/v$SNAPCRAFT_PROJECT_VERSION/brave-browser_$SNAPCRAFT_PROJECT_VERSION_amd64.deb
-    source-type: deb
-    # Correct path to icon.
-    override-pull: |
-      snapcraftctl pull
-      rm -rf etc/cron.daily/
-      rm -rf usr/bin/brave-browser usr/bin/brave-browser-stable
-      chmod 0755 ./opt/brave.com/brave/chrome-sandbox
-      sed -i 's|Icon=brave-browser|Icon=/opt/brave.com/brave/product_logo_128\.png|g' usr/share/applications/brave-browser.desktop
-      sed -i -e '/^Actions=/d' usr/share/applications/brave-browser.desktop
-    after:
-      - desktop-gtk3
-    stage-packages:
-      - gir1.2-gnomekeyring-1.0
-      - libasound2
-      - libgconf-2-4
-      - libgl1-mesa-glx
-      - libglu1-mesa
-      - libgnome-keyring0
-      - libcap2
-      - libgcrypt20
-      - libnotify4
-      - libnspr4
-      - libnss3
-      - libpulse0
-      - libxtst6
-      - libxss1
-
+base: core18
 apps:
   brave:
     command: bin/desktop-launch $SNAP/opt/brave.com/brave/brave-browser
     desktop: usr/share/applications/brave-browser.desktop
-    # Correct the TMPDIR path for Chromium Framework/Electron to
-    # ensure libappindicator has readable resources.
     environment:
-      # Correct the TMPDIR path for Chromium Framework/Electron to
-      # ensure libappindicator has readable resources
-      TMPDIR: $XDG_RUNTIME_DIR
-      # Fallback to XWayland if running in a Wayland session.
       DISABLE_WAYLAND: 1
+      BRAVE_DESKTOP: brave-browser.desktop
+      BRAVE_CONFIG_HOME: $SNAP_USER_COMMON
     plugs:
-      - avahi-observe
-      - browser-sandbox
+      - audio-playback
+      - audio-record
+      - bluez
       - camera
       - cups-control
       - desktop
       - gsettings
       - home
+      - joystick
       - mount-observe
       - network
+      - network-manager
       - opengl
       - password-manager-service
       - pulseaudio
+      - raw-usb
+      - removable-media
       - screen-inhibit-control
+      - u2f-devices
       - unity7
       - upower-observe
       - x11
-
+    slots:
+      - mpris
 plugs:
   browser-sandbox:
```

Comment:
I don't know how Snaps work, but I noticed that `browser-sandbox` has been removed in the previous section.

Reply:
From what I understand - it's not needed as the plug is defined later on globally.

See the `plug` section for the [Snapcraft.yaml reference](https://snapcraft.io/docs/snapcraft-yaml-reference) - it says:
> These plugs apply to all apps